File: C/functions/max.c
#include <stdio.h>

int Maximum(int a, int b) {
  if(a > b) {
    return a;
  }
  else {
    return b;
  }
}

int main() {
  int a = 10, b = 89;
  printf("The max is: %d", Maximum(a, b));
  return 0;
}

File: Java/max.java
class max {
    public static void main(String[] args) {
        int[] arr = {5, 9, 2, 4, 7};
        int max = arr[0];
        for (int i = 1; i < arr.length; i++) {
          if (arr[i] > max) {
            max = arr[i];
          }
        }
        System.out.println("The maximum value in the array is: " + max);
    }
}
